src/main/java/net/wasdev/wlp/common/plugins/util/DevUtil.java

```diff
@@ -417,6 +418,13 @@ public void run() {
             long timeout = verifyTimeout * 1000;
             long endTime = System.currentTimeMillis() + timeout;
 
+            // Check for port already in use error
+            String portError = serverTask.waitForStringInLog(PORT_IN_USE_MESSAGE_REGEXP, timeout, messagesLogFile);
```

Comment:
If the port is not actually in use, would this cause it to wait the entire timeout duration before it continues to check whether the server is started?